REFERENCE.md

```diff
@@ -10,11 +10,11 @@
 
 ### Functions
 
-* [`conjur::secret`](#conjursecret): Function to retrieve a Conjur secret
+* [`conjur::secret`](#conjur--secret): Function to retrieve a Conjur / DAP secret
```

Comment:
Why are we adding DAP back in here?

Reply:
Hmm good question. This file is autogenerated by the script `./ci/gen-docs.sh` which I'm guessing pulls this description from the function headers. I can go through and update the sources to only reference `Conjur` in these cases